### ISSUE TITLE
Fatal error on activate: Parse error: syntax error, unexpected

### ISSUE THREAD

**Issue by icenifields** [humanmade/comment-popularity#120]
Just tried to download the plugin on WP and activate it. Fatal error encountered.

Error "**Sytax error, unexpected '[' in /home/xxwebsitenamexx/public_html/wp-content/plugins/comment-popularity/inc/class-comment-popularity.php on line 654"**

WP is running the latest version. 4.8.2

Thanks.

**Comment by pdewouters** [comment]
Thanks, it's because I used the short array syntax `[]` instead of `array()` out of habit. I guess you are on an older PHP version that doesn't support it. I will release an update.

**Comment by icenifields** [comment]
Thanks very much for looking into this (and for the plugin itself, obviously). Sadly, I reinstalled and I'm getting further errors "Plugin could not be activated because it triggered a fatal error.... Parse error: syntax error, unexpected '[' in /home/xxxwebsitenamexx/public_html/wp-content/plugins/comment-popularity/inc/class-comment-popularity.php on line 666.
Apart from the scary line number, this looks similar to the previous bug.
I'm not sure which version of PHP I'm on, but the website is only a couple of years old, it's on the latest WP version and has the 2017 theme running. I did try searching for the PHP version but couldn't find it (sorry).
Don't worry if I'm unique in this issue. I just thought it was best to report it. Thanks again - M
